docs/markdown/Dependencies.md

```diff
@@ -653,12 +653,19 @@ or as an OSX framework.
 
 *(added 0.51.0)*
 
-Shaderc currently does not ship with any means of detection.
-Nevertheless, Meson can try to detect it using `pkg-config`, but will
-default to looking for the appropriate library manually. If the
-`static` keyword argument is `true`, `shaderc_combined` is preferred.
-Otherwise, `shaderc_shared` is preferred. Note that it is not possible
-to obtain the shaderc version using this method.
+Meson will first attempt to find shaderc using `pkg-config`. Upstream
+currently ships three different `pkg-config` files and by default will
+check them in this order: `shaderc.pc`, `shaderc_combined.pc`, and
+`shaderc_static.pc`. If the `static` keyword argument is `true`, then
+Meson instead checks in this order: `shaderc_combined`, `shaderc_static`,
```

Comment:
These two are the only places in the paragraph that don't include `.pc`, which is inconsistent.

(Although maybe they are redundant everywhere else.)

Reply:
Yup, I agree it's redundant.